src/events/onInteraction.ts

```diff
@@ -1,17 +1,30 @@
 import { Interaction, TextChannel } from 'discord.js';
 import { CommandList } from '../commands';
 import { log_channel } from '../settings.json';
+import { readFileSync } from 'fs';
+
+const autoresponses = JSON.parse(readFileSync('./autoresponses.json', 'utf-8'));
 
 export const onInteraction = async (interaction: Interaction) => {
-  if (!interaction.isCommand()) return;
-  for (const Command of CommandList) {
-    if (interaction.commandName === Command.data.name) {
-      if (interaction.commandName === 'mod' || interaction.commandName === 'bulkunban') {
-        const logChannel = interaction.guild?.channels.cache.get(log_channel) as TextChannel;
-        logChannel && logChannel.send(`${interaction.user.tag} used **${interaction.commandName}**!`);
-      }
-      await Command.run(interaction);
-      break;
+    if (!interaction.isCommand()) {
+        if ('content' in interaction) {  
+            for (const keyword in autoresponses) {
+                if (interaction.content.includes(keyword)) {
+                    await interaction.reply(autoresponses[keyword]);
+                    return; 
+                }
+            }
+        }
+        return; 
```

Comment:
Rather than this constantly checking all the messages for keywords in the json file, should add a configurable command prefix such as `!` that will trigger the autoresponse.

Reply:
Gotcha, think I misunderstood what we were looking for as far as "auto" response.